watcher/lib/influxdb/rollups.go

```diff
@@ -125,7 +169,7 @@ func buildFunction(f Function) (string, error) {
 }
 
 // isCompositeFunc checks if the specified function is composite
-func isCompositeFunc(f Function) bool {
+func (f *Function) isCompositeFunc() bool {
```

Comment:
nit: since this is already a function, `isComposite` would have been sufficient.